dev-packages/browser-integration-tests/suites/public-api/startTransaction/circular_data/test.ts

```diff
@@ -17,12 +17,12 @@ sentryTest('should be able to handle circular data', async ({ getLocalTestPath,
 
   expect(eventData.contexts).toMatchObject({
     trace: {
-      data: { lays: { contains: '[Circular ~]' } },
```

Comment:
Hah, this started failing because it depended on no other data being added to the transaction - but now under the hood it adds sample rate attribute, which lead to this not being the same at the root and one level deeper nesting. Making this an explicit data entry makes this test clearer IMHO.